internal/hud/model/model.go

```diff
@@ -0,0 +1,42 @@
+package model
+
+import (
+	"time"
+
+	"github.com/windmilleng/tilt/internal/model"
+)
+
+type Pod struct {
+	Name      string
+	StartedAt time.Time
+	Status    string
+}
+
+type Resource struct {
+	DirectoryWatched   string
+	LatestFileChanges  []string
+	LastFileChangeTime time.Time
+	Status             ResourceStatus
+}
+
+type ResourceStatus int
+
+const (
+	ResourceStatusUnbuilt ResourceStatus = iota
+	ResourceStatusStale
+	ResourceStatusFresh
+)
+
+func NewResource(directoryWatched string) Resource {
+	return Resource{
+		DirectoryWatched:   directoryWatched,
+		LatestFileChanges:  []string{},
+		LastFileChangeTime: time.Now(),
+		Status:             ResourceStatusUnbuilt,
+	}
+}
+
+type Model struct {
+	Resources map[model.ManifestName]*Resource
```

Comment:
doesn't need to happen in this PR, but this is gonna need some concurrency protection o_0 (or we can do [what DB did in his example code](https://github.com/windmilleng/tilt/blob/dbentley/watch_model/internal/state/states/store.go) and give it a channel that it listens on for updates...)

Reply:
yeah, hoping to end up with the latter